pkgs/top-level/all-packages.nix

```diff
@@ -36990,6 +36990,8 @@ with pkgs;
 
   cups-filters = callPackage ../misc/cups/filters.nix { };
 
+  libcupsfilters = callPackage ../misc/cups/libcupsfilters.nix { };
```

Comment:
Usually sorted by alphabet: (applies to multiple packages)
```nix
/* Inside the categories packages are roughly sorted by alphabet, but
 * strict sorting has been long lost due to merges […]
 ```

Reply:
I wasn't quite sure about this since the `MISC` sections seems to be a bit of a free-for-all - I couldn't figure out where to put the entries yesterday. Looking at it again, I think I might have an idea where to put them.

Reply:
I moved the printer applications, but left `libcupsfilters` and `libppd` where they are since they are very related and there seems to be precedent in the misc/CUPS section (`foomatic-db`, `gutenprint` etc.)

Reply:
> a bit of a free-for-all

that's a very adept description (: